packages/cf-wallet.js/src/app-instance.ts

```diff
@@ -17,11 +17,28 @@ export class AppInstance {
    */
   readonly id: AppInstanceID;
 
+  // Application-specific fields
   readonly appDefinition: Address;
   readonly abiEncodings: AppABIEncodings;
+  readonly timeout: BigNumber;
+
+  // Funding-related fields
   readonly myDeposit: BigNumber;
   readonly peerDeposit: BigNumber;
-  readonly timeout: BigNumber;
+
+  readonly twoPartyOutcomeInterpreterParams?: {
+    // Derived from:
+    // packages/contracts/contracts/interpreters/TwoPartyEthAsLump.sol#L10
+    playerAddrs: [string, string];
```

Comment:
It's kind of funny how we still don't have one definitive word for parties of a channel (thought of this because not all apps will be games i.e. have a notion of `players`)

Reply:
This is different, though, this is a custom parameter for the `TwoPartyOutcome` interpreter where the outcomes are basically WIN_A, WIN_B or DRAW, so "player" makes more sense. The term we agreed on at the channel level is "participant". See: https://github.com/statechannels/SCIPs/blob/master/TERMINOLOGY.md#participant